src/test/java/htsjdk/samtools/CRAMIndexQueryTest.java

```diff
@@ -94,27 +92,19 @@ public void createLocallyGeneratedCRAIFiles() throws IOException {
         cramQueryTestEmptyWithLocalCRAI.deleteOnExit();
         createLocalCRAMAndCRAI(
                 cramQueryTestEmptyWithBAI,
-                cramQueryReference,
                 cramQueryTestEmptyWithLocalCRAI,
                 tempCRAIOut
         );
     }
 
     private void createLocalCRAMAndCRAI(
             final File inputCRAM,
-            final File reference,
             final File outputCRAM,
             final File outputCRAI) throws IOException
     {
         Files.copy(inputCRAM.toPath(), outputCRAM.toPath(), StandardCopyOption.REPLACE_EXISTING);
-
-        SAMFileHeader samHeader = null;
-        try (SamReader reader = SamReaderFactory.makeDefault().referenceSequence(reference).open(inputCRAM)) {
-            samHeader = reader.getFileHeader();
-        }
         try (FileOutputStream bos = new FileOutputStream(outputCRAI)) {
-            CRAMCRAIIndexer craiIndexer = new CRAMCRAIIndexer(bos, samHeader);
```

Comment:
we don't need to construct a `CRAMCRAIIndexer` instance, so we don't need the rest either